eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -836,12 +838,11 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
       //      we did not send a shutdown message
       //        there are pending signed htlcs    => go to SHUTDOWN
       //        there are no htlcs                => go to NEGOTIATING
-
       if (!Closing.isValidFinalScriptPubkey(remoteScriptPubKey)) {
         handleLocalError(InvalidFinalScript(d.channelId), d, Some(remoteShutdown))
       } else if (Commitments.remoteHasUnsignedOutgoingHtlcs(d.commitments)) {
         handleLocalError(CannotCloseWithUnsignedOutgoingHtlcs(d.channelId), d, Some(remoteShutdown))
-      } else if (Commitments.localHasUnsignedOutgoingHtlcs(d.commitments)) { // do we have unsigned outgoing htlcs?
+      } else if (Commitments.localHasUnsignedOutgoingHtlcs(d.commitments) || Commitments.localHasUnsignedUpdateFee(d.commitments)) { // do we have unsigned outgoing htlcs?
```

Comment:
Nit: comments below should be updated to replace "unsigned htlcs" by "unsigned changes"

Reply:
Done in https://github.com/ACINQ/eclair/pull/1661/commits/200c2c03f98348bb8387359bdad91641f0d0998d